package/src/components/Message/Message.tsx

```diff
@@ -74,16 +75,25 @@ export type FileAttachmentTouchableHandlerPayload<
   additionalInfo?: { attachment?: Attachment<StreamChatGenerics> };
 };
 
+export type GalleryThumbnailTouchableHandlerPayload = {
+  emitter: 'gallery';
+  additionalInfo?: { thumbnail?: Thumbnail };
```

Comment:
I would extract it to a different type and use it here so that extending would be a easier task in future if needed.

Reply:
Not sure what you mean ? It is a different type

Reply:
added